packages/gatsby-plugin-postcss/src/gatsby-node.js

```diff
@@ -0,0 +1,99 @@
+import resolve from "./resolve"
+
+const CSS_PATTERN = /\.css$/
+const MODULE_CSS_PATTERN = /\.module\.css$/
+
+const isBuiltInCssRule = rule =>
+  rule.test &&
+  (rule.test.toString() === CSS_PATTERN.toString() ||
+    rule.test.toString() === MODULE_CSS_PATTERN.toString())
+
+const getOptions = pluginOptions => {
+  const options = { ...pluginOptions }
+
+  delete options.plugins
+
+  const postcssPlugins = options.postCssPlugins
```

Comment:
@jquense 

> maybe `postcssPlugins` to be really clear?

Yes, it is clear.

For now, I decided to use `postCssPlugins`, because this naming convention is used by many existing plugins (i.e. `gatsby-plugin-less`, `gatsby-plugin-sass` etc.).

From my point of view it is better to use `postcss Plugins` and let's discuss that later on in a new issue.